examples/bri-3/src/bri/zeroKnowledgeProof/capabilities/verifyCCSMAnchorHash/verifyCCSMAnchorHashCommand.handler.ts

```diff
@@ -0,0 +1,44 @@
+import { NotFoundException } from '@nestjs/common';
+import { CommandHandler, ICommandHandler } from '@nestjs/cqrs';
+import { CCSMAnchorHashAgent } from '../../agents/ccsmAnchorHash.agent';
+import { CCSMAnchorHashStorageAgent } from '../../agents/ccsmAnchorHashStorage.agent';
+import { VerifyCCSMAnchorHashCommand } from './verifyCCSMAnchorHash.command';
+import { CCSM_ANCHOR_HASH_NOT_FOUND_ERR_MESSAGE } from '../../api/err.messages';
+@CommandHandler(VerifyCCSMAnchorHashCommand)
+export class VerifyCCSMAnchorHashCommandHandler
+  implements ICommandHandler<VerifyCCSMAnchorHashCommand>
```

Comment:
interface?

Reply:
I thought we are not using interfaces anymore

Reply:
Just as a convention, we only removed interfaces in the models to be able to use automapper. Otherwise, we haven't restricted the use of interfaces. Also, nestjs docs require the use of the ICommandHandler interface when writing commands. https://github.com/nestjs/cqrs/blob/master/src/decorators/command-handler.decorator.ts